### ISSUE TITLE
Metadata registered in XML is reformatted in json so all words start with upper case letters

### ISSUE THREAD

**Issue by MaryHirsch** [datacite/datacite#1356]
#### Expected Behaviour

The format of the characters in the creator metadata attribute is exactly the same in xml and json

#### Current Behaviour

In XML the creator is registered as <creatorName>"This is a test"</creatorName> with only upper case for the first letter, it becomes 
name | "This Is A Test" is JSON. Having all the works starting with upper case letters is not necessarily always the desired option

#### Steps to Reproduce

1. see creators in xml here: https://doi.test.datacite.org/dois/10.17596%2Fk75b-4c94
2. see creators in json here: https://api.test.datacite.org/dois/10.17596/k75b-4c94

### Context (Environment)

Production

#### Possible Implementation

<img src="https://frontapp.com/assets/img/favicons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_2oolx)

**Comment by mpaluch** [comment]
Hello,

This issue is affecting all of the NRC.IR metadata and is most prevalent in French language records such as the examples below. Our expectation is that metadata contents are stored and displayed verbatim and that any citations generated for the records will match the records themselves.


### [10.4224/cy0b-kg50](https://doi.org/10.4224/cy0b-kg50)
Original submitted [DCMS metadata](https://api.datacite.org/dois/application/vnd.datacite.datacite+xml/10.4224/cy0b-kg50):
```
<creator>
    <creatorName nameType="Organizational" xml:lang="fr">Commission canadienne des codes du bâtiment et de prévention des incendies</creatorName>
</creator>
```

DataCite [JSON](https://api.datacite.org/dois/10.4224/cy0b-kg50):
```
"creators":[ 
  {
    "name":"Commission Canadienne Des Codes Du Bâtiment Et De Prévention Des Incendies",
    "nameType":"Organizational",
    "affiliation":[],
    "nameIdentifiers":[]
  }
]
```

Citation:
![image](https://user-images.githubusercontent.com/427055/160451311-58ef324b-0636-49fb-b090-ec40529b07b3.png)


### [10.4224/40002663](https://doi.org/10.4224/40002663)
Original submitted [DCMS metadata](https://api.datacite.org/dois/application/vnd.datacite.datacite+xml/10.4224/40002663):
```
<creator>
  <creatorName nameType="Organizational" xml:lang="fr">l’Institut de prévention des sinistres catastrophiques</creatorName>
</creator>
```

DataCite [JSON](https://api.datacite.org/dois/10.4224/40002663):
```
{
 "name":"L’Institut De Prévention Des Sinistres Catastrophiques",
 "nameType":"Organizational",
 "affiliation":[],
 "nameIdentifiers":[]
}
```

Citation:
![image](https://user-images.githubusercontent.com/427055/160450954-43064e1d-9a87-442a-952a-6f26ba35393a.png)

**Comment by richardhallett** [comment]
I had an initial look and the fix is in bolognese, but the problem is, this was decided at some point to always "titleize" the author names and has done this since 2017. 
However I am not sure entirely why this was the case, there might be other edge cases beyond the above example where we do want to try and make the author name look a bit better.

For reference the code that does this: https://github.com/datacite/bolognese/blob/master/lib/bolognese/author_utils.rb#L102
And changing this would break a few tests, possibly in lupo too.

My personal thoughts on this is that we shouldn't try and do this ever and just leave the metadata untouched, but I think a bit more consideration before changing might be prudent.

@kjgarza Any thoughts on this?

**Comment by jrhoads** [comment]
I agree.  We shouldn't be reformatting the metadata entered by the user.  Especially in names.

**Comment by codycooperross** [comment]
This should now be fixed by https://github.com/datacite/bolognese/pull/204
